package-lock.json

```diff
@@ -141,6 +141,22 @@
         "workbox-webpack-plugin": "6.5.0"
       }
     },
+    "../ensembl-genome-browser": {
+      "name": "@ensembl/ensembl-genome-browser",
+      "version": "0.2.1-test.2",
+      "extraneous": true,
+      "license": "ISC",
+      "devDependencies": {
+        "@rollup/plugin-typescript": "8.2.1",
+        "@typescript-eslint/eslint-plugin": "4.28.1",
+        "@typescript-eslint/parser": "4.28.1",
+        "eslint": "7.29.0",
+        "rollup": "2.52.6",
+        "rollup-plugin-copy": "3.4.0",
+        "tslib": "2.3.1",
+        "typescript": "4.3.4"
+      }
+    },
```

Comment:
This whole block was created when you were building ensembl-genome-browser locally, and shouldn't be committed to our dev.

Reply:
Thanks for pointing this out. Also noticed, the `.../ensembl-genome-browser` is strangely properly resolved yet. Will look into this again.

Reply:
Removed it ✅